oe_editorial.module

```diff
@@ -6,3 +6,50 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function oe_editorial_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+  /** @var \Drupal\Core\Field\FieldDefinitionInterface $field_definition */
+  $field_definition = $context['items']->getFieldDefinition();
+
+  // We add an after build method so we can alter the Url of the help link.
+  if (in_array($field_definition->getType(), [
+    'text',
+    'text_long',
+    'text_with_summary',
+  ])) {
+    $element['#after_build'][] = '_oe_editorial_alter_textarea_help';
+  }
+}
+
+/**
+ * After build callback to alter the Url of the help link on text areas.
+ */
+function _oe_editorial_alter_textarea_help($form_element, FormStateInterface $form_state) {
+  if (isset($form_element['format']['format'])) {
```

Comment:
Could we start this as: 
if (!isset($form_element['format']['format'])) {
  return;
}
...